policyengine_us/tests/policy/contrib/reconciliation/ctc_ssn.yaml

```diff
@@ -0,0 +1,86 @@
+- name: Case 1, CTC higher than 0.
+  period: 2026
+  absolute_error_margin: 0.9
+  reforms: policyengine_us.reforms.reconciliation.ctc_ssn.ctc_ssn
+  input:
+    gov.contrib.reconciliation.pease.in_effect: true
```

Comment:
correct parameter?

Reply:
Initializing the reform seems to bypass the parameter value but changed for posterity